README.md

```diff
@@ -65,7 +64,7 @@ following steps:
    happens the updated entity metadata will be available for queries.
 
 <!-- markdownlint-disable line-length -->
-[`oasis-registry`]: https://github.com/oasisprotocol/metadata-registry-tools
+[`oasis-cli`]: https://github.com/oasisprotocol/cli
```

Comment:
Could also link to CLI [docs](https://docs.oasis.io/general/manage-tokens/cli/)?